BondageClub/Assets/Female3DCG/Female3DCG.js

```diff
@@ -4104,7 +4104,12 @@ var AssetFemale3DCG = [
 					{ Name: "Text" },
 				]
 			},
-			{ Name: "ServingTray", Value: -1, Time: 5, Extended: true },
+			{ Name: "ServingTray", Value: -1, Time: 5, Extended: true, Layer: [
+					{ Name: "Tray", AllowColorize: false },
+					{ Name: "Objects" },
```

Comment:
It's generally preferable to use `AllowTypes` on the layers to remove the need for empty placeholder assets. That makes it so that the layer is only rendered when the current type is in the `AllowTypes` array.
```suggestion
					{ Name: "Objects", AllowTypes: ["Drinks", "Cake", "Cookies", "Toys"] },
```
This means the "Objects" layer would not be rendered for the empty type, and the `ServingTray_Objects.png` asset could be removed. Similarly for the "Details" layer.

Reply:
Thanks a lot!